regtests/bin/matrix_cmake_ncep

```diff
@@ -62,20 +66,18 @@ EOF
     batchq='slurm'
     basemodcomp='intel/2022.1.2'
     basemodmpi='impi/2022.1.2'
-    hpcstackpath='/scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/intel-2022.1.2/modulefiles/stack'
-    hpcstackversion='hpc/1.2.0'
-    modcomp='hpc-intel/2022.1.2'
-    modmpi='hpc-impi/2022.1.2'
-    scotchpath='/scratch1/NCEPDEV/climate/Matthew.Masarik/waves/opt/hpc-stack/scotch-v7.0.3/install'
-    metispath='/scratch1/NCEPDEV/climate/Matthew.Masarik/waves/opt/hpc-stack/parmetis-4.0.3/install'
-    modcmake='cmake/3.20.1'
+    spackstackpath='/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.0/envs/unified-env-noavx512/install/modulefiles/Core'
+    modcomp='stack-intel/2021.5.0'
+    modmpi='stack-intel-oneapi-mpi/2021.5.1'
+    metispath='/scratch1/NCEPDEV/climate/Matthew.Masarik/waves/opt/spack-stack/1.5.0/parmetis-4.0.3/install'
+    modcmake='cmake/3.23.1'
   elif [ $isorion ]
   then
     batchq='slurm'
     basemodcomp='intel/2022.1.2'
     basemodmpi='impi/2022.1.2'
-    hpcstackpath='/work/noaa/epic-ps/role-epic-ps/hpc-stack/libs/intel-2022.1.2/modulefiles/stack'
-    hpcstackversion='hpc/1.2.0'
+    spackstackpath='/work/noaa/epic-ps/role-epic-ps/hpc-stack/libs/intel-2022.1.2/modulefiles/stack'
```

Comment:
What about updating to this: https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_orion.intel.lua#L5 ?